exporter/api.py

```diff
@@ -37,8 +38,30 @@ def sanitize(title):
                 id=learning_resource.id,
                 title=sanitize(learning_resource.title),
             )
-            with open(os.path.join(tempdir, filename), 'w') as f:
+            type_name = learning_resource.learning_resource_type.name
+            try:
+                os.mkdir(os.path.join(tempdir, type_name))
+            except OSError:
+                pass
+            with open(os.path.join(tempdir, type_name, filename), 'w') as f:
                 f.write(learning_resource.content_xml)
+
+            for static_asset in learning_resource.static_assets.all():
+                # Output to static directory.
+                static_path = re.sub(r"^assets/", "static/",
+                                     static_asset.asset.name)
+
+                relative_path, filename = os.path.split(static_path)
+                try:
+                    os.makedirs(os.path.join(tempdir, relative_path))
+                except OSError as ex:
+                    # Directory already exists
+                    if "File exists" not in ex.args:
+                        raise
```

Comment:
If the directory already exists, it's OK to pass, otherwise we should reraise the exception